R/variables.R

```diff
@@ -240,15 +243,13 @@ Probabilities.GradientBoost <- function(object)
     if(object$numeric.outcome)
         stop("Probabilities are only applicable to models with categorical outcome variables.")
 
-    data <- CheckPredictionVariables(object, object$model)
-    data <- OneHot(data, object$outcome.name)$X
```

Comment:
These lines are not needed since the equivalent execution is done in https://github.com/Displayr/flipMultivariates/pull/8/files#diff-96019631575c4d9c147e901dcc2611fbL194-L195